PWGLF/DataModel/LFStrangenessTables.h

```diff
@@ -145,6 +145,9 @@ DECLARE_SOA_COLUMN(DCANegToPV, dcanegtopv, float);         //! DCA negative pron
 DECLARE_SOA_COLUMN(V0CosPA, v0cosPA, float);               //! V0 CosPA
 DECLARE_SOA_COLUMN(DCAV0ToPV, dcav0topv, float);           //! DCA V0 to PV
 
+// Type of V0 from the svertexer (photon, regular, from cascade)
+DECLARE_SOA_COLUMN(V0Type, v0Type, uint8_t); //! DCA V0 to PV
```

Comment:
The description is not correct.

Reply:
I am not even sure if this will then propagate to the documentation.

Reply:
So the autogenerated doc problem is currently being investigated by @pbuehler (super thanks!) - there seem to be some sophisticated new preprocessor-defined tables in the framework that complicate the auto-generation. Once that's sorted out I think these comments do in fact go into the docs... I'll keep an eye :-D